Cargo.toml

```diff
@@ -29,9 +29,9 @@ dmsort = "1.0.1"
 
 [features]
 default=["vec-id-sets"]
-# Enable the parallel runtime implementation todo make default
+# Enable the parallel runtime implementation # TODO make default
```

Comment:
This looks a little weird. Also, I'm not sure I understand what's left to do.

Reply:
The idea was to make the `parallel-runtime` feature a default feature of the crate. Currently if you want parallel execution you need to enable it explicitly with a lot of ceremony: https://github.com/lf-lang/lingua-franca/blob/4f8d8893574e3c76fd30c2515c9be9a7444e4f1f/test/Rust/src/target/CargoDependencyOnRuntime.lf#L5